runtime/commonMain/src/kotlinx/serialization/json/JsonConfiguration.kt

```diff
@@ -6,37 +6,60 @@ package kotlinx.serialization.json
 
 import kotlinx.serialization.UnstableDefault
 import kotlinx.serialization.UpdateMode
-import kotlin.jvm.JvmField
-import kotlin.jvm.JvmStatic
+import kotlin.jvm.*
 
 /**
  * The class responsible for JSON-specific customizable behaviour in [Json] format.
  *
  * Options list:
  * * [encodeDefaults] specifies whether default values are encoded.
- * * [strictMode] enables strict mode, which prohibits unknown keys in the JSON and non-numerical values in floating point numbers.
- * * [unquoted] specifies whether keys and values should be quoted.
+ *
+ * * [ignoreUnknownKeys] ignores encounters of unknown properties in the input JSON.
+ *
+ * * [isLenient] removes JSON specification restriction (RFC-4627) and makes parser
+ *   more liberal to the malformed input. In lenient mode quoted string literals,
+ *   and unquoted string literals are allowed.
+ *
+ * * [serializeSpecialFloatingPointValues] removes JSON specification restriction on
+ *   special floating-point values such as `NaN` and `Infinity` and enables their
+ *   serialization. When enabling it, please ensure that the receiving party will be
+ *   able to parse these special value.
```

Comment:
```suggestion
 *   able to parse these special values.
```

(or `this special value`)